mrmustard/lab_dev/states/coherent.py

```diff
@@ -17,7 +17,7 @@
 """
 
 from __future__ import annotations
-
+from typing import Iterable
```

Comment:
Not to be nitpicky, but I think these should be `Sequence`, because `Iterable` can be unordered (like `set`).